boot/freeldr/freeldr/CMakeLists.txt

```diff
@@ -252,91 +128,10 @@ list(APPEND FREELDR_BASE_SOURCE
-    add_linker_script(freeldr_pe freeldr_gcc.lds)
-    # Strip everything, including rossym data
-    add_custom_command(TARGET freeldr_pe
-                    POST_BUILD
-                    COMMAND ${CMAKE_STRIP} --remove-section=.rossym $<TARGET_FILE:freeldr_pe>
-                    COMMAND ${CMAKE_STRIP} --strip-all $<TARGET_FILE:freeldr_pe>)
-endif()
-
-set_image_base(freeldr_pe 0x10000)
-set_subsystem(freeldr_pe native)
-set_entrypoint(freeldr_pe RealEntryPoint)
-
-if(ARCH STREQUAL "i386")
-    target_link_libraries(freeldr_pe mini_hal)
-endif()
-
-target_link_libraries(freeldr_pe freeldr_common cportlib blcmlib blrtl libcntpr)
-
-# dynamic analysis switches
-if(STACK_PROTECTOR)
-    target_sources(freeldr_pe PRIVATE $<TARGET_OBJECTS:gcc_ssp_nt>)
-endif()
-
-if(RUNTIME_CHECKS)
-    target_link_libraries(freeldr_pe runtmchk)
-    target_link_options(freeldr_pe PRIVATE "/MERGE:.rtc=.text")
-endif()
-
-add_dependencies(freeldr_pe asm)
-
-if(SARCH STREQUAL "pc98")
-    file(MAKE_DIRECTORY ${REACTOS_BINARY_DIR}/PC98)
-    add_custom_target(pc98bootfdd
-        COMMAND native-fatten ${REACTOS_BINARY_DIR}/PC98/ReactOS-98.IMG -format 2880 ROS98BOOT -boot ${CMAKE_BINARY_DIR}/boot/freeldr/bootsect/pc98/fat12fdd.bin -add ${CMAKE_CURRENT_BINARY_DIR}/freeldr.sys FREELDR.SYS -add ${CMAKE_SOURCE_DIR}/boot/bootdata/floppy_pc98.ini FREELDR.INI
-        DEPENDS native-fatten fat12pc98 freeldr
-        VERBATIM)
-endif()
-
-if(NOT ARCH STREQUAL "arm")
-    concatenate_files(
-        ${CMAKE_CURRENT_BINARY_DIR}/freeldr.sys
-        ${CMAKE_CURRENT_BINARY_DIR}/frldr16.bin
-        ${CMAKE_CURRENT_BINARY_DIR}/$<TARGET_FILE_NAME:freeldr_pe>)
-    add_custom_target(freeldr ALL DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/freeldr.sys)
-else()
-    add_custom_target(freeldr ALL DEPENDS freeldr_pe)
-endif()
-
-# Rename freeldr on livecd to setupldr.sys because isoboot.bin looks for setupldr.sys
-add_cd_file(TARGET freeldr FILE ${CMAKE_CURRENT_BINARY_DIR}/freeldr.sys DESTINATION loader NO_CAB FOR bootcd regtest)
-add_cd_file(TARGET freeldr FILE ${CMAKE_CURRENT_BINARY_DIR}/freeldr.sys DESTINATION loader NO_CAB NOT_IN_HYBRIDCD FOR livecd hybridcd NAME_ON_CD setupldr.sys)
-
-if(NOT ARCH STREQUAL "arm")
-    concatenate_files(
-        ${CMAKE_CURRENT_BINARY_DIR}/setupldr.sys
-        ${CMAKE_CURRENT_BINARY_DIR}/frldr16.bin
-        ${CMAKE_CURRENT_BINARY_DIR}/$<TARGET_FILE_NAME:freeldr_pe>)
-    add_custom_target(setupldr ALL DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/setupldr.sys)
-else()
-    add_custom_target(setupldr ALL DEPENDS freeldr_pe)
-endif()
+# if(ARCH STREQUAL "i386")
+#     # Must be included together with disk/scsiport.c
+#     list(APPEND FREELDR_BASE_SOURCE
+#         ${CMAKE_CURRENT_BINARY_DIR}/freeldr_pe.def)
+# endif()
 
-add_cd_file(TARGET setupldr FILE ${CMAKE_CURRENT_BINARY_DIR}/setupldr.sys DESTINATION loader NO_CAB FOR bootcd regtest)
+include(pcat.cmake)
```

Comment:
``pcat.cmake`` filename does not fit well, because it includes PC-98 which is not PC/AT compatible, and even some ARM stuff; Better name it ``legacy.cmake`` or ``lgcyboot.cmake`` (contrary to UEFI boot)

Reply:
True. Well (apart from ARM which is here "legacy" because it doesn't use uefi and could be in its own file as well), the pc-98 and xbox are bios-based architectures.

Reply:
> For info, I chose "pcat" also because this is the name given to windows bootmgr when it's not efi

In Windows they switched to bootmgr since Vista, and it never supported anything besides PC/AT, so it makes sense for the bootmgr; But our FreeLoader is meant to be a NTLDR replacement; Maybe `bios.cmake` then?